Basic functionality ready!

greatrobocreator opened the pull request:
+ Cesar encryption/decryption
+ Frequency analyzing
+ GUI for all of that

greatrobocreator committed:
Basic functional ready!

+ Cesar encryption/decryption
+ Frequency analyzing
+ GUI for all of that

febos commented on Analyzer.py:
```diff
@@ -0,0 +1,26 @@
+
+    def analyze(self, text):
+        alphabet = [chr(i + 65) for i in range(26)] + [chr(i + 97) for i in range(26)]
+        frequency = [0] * 26
```
используйте константы

febos commented on main.py:
```diff
@@ -0,0 +1,152 @@
+import PySimpleGUI as sg
+import matplotlib.pyplot as plt
+from Cypher import *
+from Analyzer import *
```
никогда не используйте импорт со звездочкой

febos reviewed (commented):
1) Не работает:
![image](https://user-images.githubusercontent.com/1099991/113409978-0a8b4980-93bb-11eb-9f65-e73f103f1442.png)
окно ни на что не реагирует (centos 7, python 3.6)

2) см. замечания к коду
3) у вас неделя на следующую итерацию

greatrobocreator committed:
Update README.md

greatrobocreator committed:
Update README.md

greatrobocreator committed:
Update README.md

greatrobocreator committed:
Update README.md

greatrobocreator committed:
Update README.md

greatrobocreator committed:
+ Open Files
+ Cesar Cracker
+ Vigenere cypher
+ Vigenere cracker!
+ Good system for inputs/output for cyphers, analyzers and crackers
+ Save Files

greatrobocreator committed:
Merge remote-tracking branch 'origin/dev' into dev

greatrobocreator committed:
+ Multiliine size increased
+ Max key length increased

greatrobocreator committed:
+ CoincidenceIndexAnalyzer added as independent tool

febos commented:
проект принят с 1 доп баллом

greatrobocreator committed:
Merge branch 'master' into dev
